src/sentry/migrations/0050_auto_20200306_0007.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-03-06 00:07
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import sentry.db.models.fields.bounded
+
+
+class Migration(migrations.Migration):
+    # This flag is used to mark that a migration shouldn't be automatically run in
+    # production. We set this to True for operations that we think are risky and want
+    # someone from ops to run manually and monitor.
+    # General advice is that if in doubt, mark your migration as `is_dangerous`.
+    # Some things you should always mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that
+    #   they can be monitored. Since data migrations will now hold a transaction open
+    #   this is even more important.
+    # - Adding columns to highly active tables, even ones that are NULL.
+    is_dangerous = False
+
+    # This flag is used to decide whether to run this migration in a transaction or not.
+    # By default we prefer to run in a transaction, but for migrations where you want
+    # to `CREATE INDEX CONCURRENTLY` this needs to be set to False. Typically you'll
+    # want to create an index concurrently when adding one to an existing table.
+    atomic = True
+
+
+    dependencies = [
+        ('sentry', '0049_auto_20200304_0254'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='querysubscription',
+            name='status',
+            field=models.SmallIntegerField(default=0),
+        ),
+        migrations.AlterField(
+            model_name='exporteddata',
+            name='query_type',
+            field=sentry.db.models.fields.bounded.BoundedPositiveIntegerField(choices=[(0, b'Discover'), (1, b'Billing Report'), (2, b'Issues-by-Tag')]),
+        ),
```

Comment:
What's this for?

Reply:
Someone didn't generate a migration after making changes to a list of choices. It causes no sql operations, just migration state changes, so I was just going to leave it in.

Have been planning for a while to have something in CI that runs `makemigrations` and ensures that it produces no output, so that this stops happening.